### ISSUE TITLE
beta3.31 IMPORTANT: Remove jQuery dependency???

### ISSUE THREAD

**Issue by AlexandreZaytsev** [uikit/uikit#2954]
UIkit version 3.0.0-beta.31

what does it mean? stopped working JScript scripts compared to the v30

**Comment by AlekVolsk** [comment]
what were your scripts? on jquery? his bole no, he cut

**Comment by AlexandreZaytsev** [comment]
navigation tree
like 
```
document.getElementById(uid1 + '_click_byDir').getElementsByClassName('uk-active')[0].getElementsByTagName('a')[0].setAttribute("href", urlObj);
...
BX.ready(function(){                                                                                     //событие по готовности модели DOM
     $('#' + paramObj.getUID() + '_hover_byDir').on('hidden', function () {               //событие перед закрытим для выбора направления
      paramObj.setHrefDOM("byDir", this.getElementsByClassName('uk-active')[0].getElementsByTagName("a")[0].text.trim()); });
...
```
etc.
and maybe not in the scripts thing...

**Comment by AlexandreZaytsev** [comment]
stop
fix error in topic [#2955 ](https://github.com/uikit/uikit/issues/2955)

**Comment by janschoenherr** [comment]
Discussion continues on #2955
